docs/clusters/quick-install.md

```diff
@@ -0,0 +1,58 @@
+---
+id: quick-install
+title: How to quickly install a Temporal Cluster for testing and local development
+sidebar_label: Quick install
+description: There are four ways to quickly install and run a Temporal Cluster.
+---
+
+There are four ways to quickly install and run a Temporal Cluster:
+
+- [Docker](#docker): Using Docker Compose makes it easy to develop your Temporal Application locally.
+- [Render](#render): Our [temporalio/docker-compose](https://github.com/temporalio/docker-compose) experience has been translated to Render's Blueprint format for a an alternative cloud connection.
+- [Helm Charts](#helm-charts): Deploying a Cluster to [Kubernetes](https://kubernetes.io/) is an easy way to test the system and develop Temporal Applications.
+- [Gitpod](#gitpod): One-click deployments are available for Go and TypeScript.
+
+**We do not recommend using any of these methods in a [full (production) environment](/docs/server/production-deployment).**
+
+### Helm Charts
+
+Use [Temporal Helm Charts](https://github.com/temporalio/helm-charts) to deploy the Temporal Server to a [Kubernetes](https://kubernetes.io/) cluster.
+
+Deploying the Temporal Cluster with Helm is not recommended for a production environment, but it is a great way to test the system while developing Workflows.
+
+### Docker Compose
+
+Use Docker Compose and Temporal Cluster Docker Images to quickly install and run a Temporal Cluster locally while developing Workflows.
```

Comment:
```suggestion
Use Docker Compose and Temporal Cluster Docker images to quickly install and run a Temporal Cluster locally while developing Workflows.
```
Lowercase "images" in "Docker images" per https://docs.docker.com/glossary/#image.